src/main/kotlin/net/atos/zac/enkelvoudiginformatieobject/model/EnkelvoudigInformatieObjectLock.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.enkelvoudiginformatieobject.model
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotBlank
+import net.atos.zac.util.FlywayIntegrator
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import java.util.UUID
+
+@Entity
+@Table(schema = FlywayIntegrator.SCHEMA, name = "enkelvoudiginformatieobject_lock")
+@SequenceGenerator(
+    schema = FlywayIntegrator.SCHEMA,
+    name = "sq_enkelvoudiginformatieobject_lock",
+    sequenceName = "sq_enkelvoudiginformatieobject_lock",
+    allocationSize = 1
+)
+@AllOpen
+@NoArgConstructor
+class EnkelvoudigInformatieObjectLock {
```

Comment:
Why not a `data class` btw?

Reply:
Because according to IntelliJ:
> The data class implementations of equals(), hashCode() and toString() are not recommended for JPA entities. They can cause severe performance and memory consumption issues.

Reply:
aha! ok clear

Reply:
in theory you could then override these functions to make them more JPA-compatible I guess but that seems a bit cumbersome indeed..